services/src/main/java/org/keycloak/protocol/oidc/OIDCAdvancedConfigWrapper.java

```diff
@@ -131,6 +131,24 @@ public void setUseMtlsHoKToken(boolean useUtlsHokToken) {
         setAttribute(OIDCConfigAttributes.USE_MTLS_HOK_TOKEN, val);
     }
 
+    public boolean isUseMtlsExtendedValidation() {
```

Comment:
IMO, it might be good if these non standard client metadata are not added to this class by accessing client's attributes directly.

It might be helpful to refer to codes for client metadata ["backchannel_token_delivery_mode"](https://github.com/keycloak/keycloak/blob/af1855ebc6a3d025b58b697c9bc32567fb512eec/server-spi/src/main/java/org/keycloak/models/CibaConfig.java#L28).

Reply:
Completely reverted to original state.